planner/core/point_get_plan.go

```diff
@@ -1405,6 +1405,11 @@ func buildOrderedList(ctx sessionctx.Context, plan Plan, list []*ast.Assignment,
 			Col:     col,
 			ColName: plan.OutputNames()[idx].ColName,
 		}
+		// TODO: Check if needed!
+		defaultExpr := extractDefaultExpr(assign.Expr)
+		if defaultExpr != nil {
+			defaultExpr.Name = assign.Column
+		}
```

Comment:
Why add this? Any examples?

Reply:
It would panic for the new test I added. I will remove the TODO line.
```
        tk.MustExec("update t2 set b = default, c = default WHERE a = 2")
```